clients/drcachesim/tracer/tracer.cpp

```diff
@@ -2770,9 +2776,9 @@ init_thread_in_process(void *drcontext)
 
     if (op_use_physical.get_value()) {
         if (!data->physaddr.init()) {
```

Comment:
Wouldn't this always return false for windows?

Reply:
Yes, which is what we want since there's no support for this flag there.